eng/pipelines/docs.yml

```diff
@@ -0,0 +1,84 @@
+trigger:
+  - GenerateDocsForGitHub
+jobs:
+  - job: CreateDocs
+    pool:
+      vmImage: vs2017-win2016
+    steps:
+      - powershell: |
+          Invoke-WebRequest -Uri "https://github.com/dotnet/docfx/releases/download/v2.43.2/docfx.zip" `
```

Comment:
Why are you downloading this way instead of using the nuget package?

Reply:
It seems easier to use it this way , as downloading from nuget results in .nupkg files rather than the needed exe.

Reply:
For this release it should be fine but we should look at using the dotnet install options later. If this is a proper tools package we can do dotnet install and then run the command, which is the more appropriate way to do this.

Reply:
Do you mean nuget install?